app/src/androidTest/java/ch/epfl/sweng/zuluzulu/MainActivityAsUserTest.java

```diff
@@ -13,6 +17,6 @@
     @Test
     public void isAuthenticated() {
         // check not authenticated
-        assertTrue(mActivityRule.getActivity().isAuthenticated());
+        onView(withId(R.id.main_user_fragment)).check(matches(isDisplayed()));
```

Comment:
Pourquoi tu as supprimé mon test ? C'était pour vérifier si isAuthenticated() retournait la bonne valeur

Il faut laisser.

Ton test tu le mets dans une autre fonction

Reply:
Sinon tu vas baisser le test coverage sur MainActivity.isAuthenticated()...

Reply:
Genre isTheUserLayoutOpen() par exemple

Reply:
ah okay je m occuper de ça

Reply:
in fact i don't even need to create two methods as both what you did and what i added test the same thing, so i can just readd what you wrote and leave both in the same method

Reply:
there change done in commit 0c11e17

Reply:
The two methods don't tests the same thing.

The first one test if isAuthanticated() is working correctly

The otherone test if the fragment layout is the right one

It's not the same thing and this should not be in the same method at my opinion

But that's okay :+1: